docs/Demo.md

```diff
@@ -1,73 +1,175 @@
+
+```
+git log -p -1
+```
+Compare the output with the following command:
+```
+curl $(oc get route -n pelorus committime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
+```
+
+Also notice the built deployed image sha is visible via the deploytime-exporter
+
+```
+curl $(oc get route -n pelorus deploytime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
 ```
 
-The script will start and will begin to run the ansible-applier on the path to container-pipelines. It will attempt to install ansible-galaxy, skipping if it is, and then run the ansible-applier playbook. This will setup a build (jenkins) as well as a dev, stage and prod environment. Once jenkins has been setup, an initial run of the basic-nginx app will build and deploy. Once that is complete, the dashboard should resemble the following:
-![First-Deploy-Data](img/pelorus-dashboard-first-deploy.png)
+### See the change to your todolist application
+You can now also check on your todolist application and see the updated text change "Add a todo"
+![todolist-fixed](img/todolist_fixed.png)
 
-A prompt will appear on the screen waiting for user input, the user can "rerun" the script. What this will do is create a change in the index.html file, git commit and push it to the url of the forked repo, then create another pipeline build. Then the second pipeline build of basic-nginx will start. Once complete, the dashboard will update again:
-![Second-Deploy-Data](img/pelorus-dashboard-second-deploy.png)
 
 ## Troubleshooting
 
-Note: If exporters are not functioning or deployed, no data will show up in the dashboard. It will look like the following:
+If exporters are not functioning or deployed, no data will show up in the dashboard. It will look like the following:
 ![No-Data](img/pelorus-dashboard-no-data.png)
 
+Check the logs of exporter pod.
+
 An "idle" state could resemble:
 ![Idle-Data](img/pelorus-dashboard-idle-data.png)
 
-Run the demo script
-``` pelorus/demo/demo.sh <path to container-pipelines> <url to forked repo>```
 
-Once the script has completed the openshift applier segment, it will pause for 5 minutes in order for jenkins to get setup and basic-nginx to run through it's first pipeline build. Once the pipeline has completed the dashboard will update accordingly:
-![First-Deploy-Data](img/pelorus-dashboard-first-deploy.png)
+## Automated Demo
+
+The Pelorus and todolist application can be installed automatically.  Using the forked copy of [mig-demo-apps](https://github.com/konveyor/mig-demo-apps) referenced as `https://github.com/<your_org>/mig-demo-apps`, execute the following steps:
+
+> **Note:**
+> The automated demo expects that Pelorus is not installed and the Pelorus namespace is *not* present.
+
+1. Setup the github webhook prior to execution
+
+2. Execute `run-pelorus-e2e-tests`
+```
+# ensure pelorus has been uninstalled completely
+cd pelorus
+make dev-env
+scripts/run-pelorus-e2e-tests -o <your_org>
```

Comment:
`<your_org>` - what is it? Could be more explicit saying that it's the org where xyz was forked, and give real example.

Secondly, when we run-pelorus-e2e-tests I think we need as well to:
```
source .venv/bin/activate
export KUBECONFIG=....
```